core/gather/driver/execution-context.js

```diff
@@ -79,9 +79,10 @@ class ExecutionContext {
    * page without isolation.
    * @param {string} expression
    * @param {number|undefined} contextId
+   * @param {boolean=} returnByValue
    * @return {Promise<*>}
    */
-  async _evaluateInContext(expression, contextId) {
+  async _evaluateInContext(expression, contextId, returnByValue = true) {
```

Comment:
I don't love the changes I made here. Any suggestions?

I need to _not_ return by value in the gatherer to get access to the backend node ids (`result.objectId` is not returned when returning `Runtime.evaluate` by value).

Reply:
> I don't love the changes I made here. Any suggestions?
> 
> 
> 
> I need to _not_ return by value in the gatherer to get access to the backend node ids (`result.objectId` is not returned when returning `Runtime.evaluate` by value).